servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -2089,6 +2089,26 @@ public void testDiff() throws BaseNessieClientServerException {
             });
   }
 
+  @Test
+  public void commitLogExtendedForUnchangedOperation() throws Exception {
+    String branch = "main";
```

Comment:
Could you use a test case-specific branch? I believe most test cases create a dedicated branch up front. That should minimize interference between tests.

Reply:
ok. done.